api/src/services/AuthService.ts

```diff
@@ -21,28 +25,54 @@ export class AuthService {
     ) {
         this.userRepository = entityManager.getRepository(User);
     }
```

Comment:
I would be really nice to inject directly the userRepository from di container instead of entityManager and then fetching from it the user repo. This way the service doesn't know the concrete instance of user repository